server/package.json

```diff
@@ -19,7 +19,17 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "db:docker:up": "docker compose up -d",
+    "db:docker:down": "docker compose down"
+  },
+  "lint-staged": {
```

Comment:
I'm getting a warning about `lint-staged` being a duplicate key, so you might need to remove the other one to get rid of that

Reply:
Oh wow, i have no idea how that happened lol

Thank you, I'll make the changes ASAP